src/shared/components/Checkbox/Checkbox.tsx

```diff
@@ -69,15 +81,22 @@ export const Checkbox = React.forwardRef<HTMLInputElement, CheckboxProps>(
               data-multiple="false"
               checked={isSelected}
               disabled={disabled}
-              onChange={onChangeHandler}
-              onFocus={onFocusHandler}
-              onBlur={onBlurHandler}
+              onChange={handleChange}
+              onFocus={handleFocus}
+              onBlur={handleBlur}
               {...props}
             />
             <Checkmark>{isSelected ? <Icon name={isIndeterminate ? 'dash' : 'check'} /> : null}</Checkmark>
           </InnerContainer>
         </Container>
-        {label && <LabelText variant="subtitle2">{label}</LabelText>}
+        <TextContainer>
+          {label && <LabelText variant="subtitle2">{label}</LabelText>}
```

Comment:
I think this should be `vartiant="body1"` as in Figma, then you don't need the additional `line-height` style